Dockerfile

```diff
@@ -1,31 +1,33 @@
-FROM python:3.6-slim
+FROM python:3.6.9-slim
```

Comment:
I like pegging at a specific Python version so that it doesn't update underneath us.